demo/webcam_demo.py

```diff
@@ -51,26 +58,38 @@ def parse_args():
 
 
 def predict_webcam_video():
-    data = dict(img_shape=None, modality='RGB')
-
     windows = deque()
     score_cache = deque()
     scores_sum = np.zeros(len(label))
 
+    if need_preparation:
+        is_prepared = False
+        print('This model needs some preparation time')
+        prog_bar = mmcv.ProgressBar(sample_length)
+
     while True:
         ret, frame = camera.read()
         # BGR to RGB
         windows.append(np.array(frame[:, :, ::-1]))
         if data['img_shape'] is None:
             data['img_shape'] = frame.shape[:2]
 
+        if need_preparation and not is_prepared:
+            prog_bar.update()
+            if len(windows) != sample_length:
```

Comment:
how about fill the whole buffer with the first frame, so that no preparation is needed?

Reply:
Done.